bodybank/bodybank/settings.py

```diff
@@ -39,7 +39,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'accounts'
+    'accounts',
+    'bootstrap5'
```

Comment:
I usually put 3rd party before my own apps, but later learned that this hardly makes a difference